docs/release-notes/changelog.rst

```diff
@@ -3,6 +3,102 @@
 2.x Changelog
 =============
 
+.. changelog:: 2.7.1
+    :date: 2024-03-22
+
+    .. change:: add default encoders for `Enums` and `EnumMeta`
+        :type: bugfix
+        :pr: 3193
+
+        This addresses an issue when serializing `Enums` that was reported in discord.
+
+    .. change:: replace TestClient.__enter__ return type with Self
```

Comment:
should we use double backticks here? I see some more single backticks, ignore these if I am wrong